src/Wavefront.OpenTracing.SDK.CSharp/Wavefront.OpenTracing.SDK.CSharp.csproj

```diff
@@ -30,9 +30,9 @@
   <ItemGroup>
     <PackageReference Include="OpenTracing" Version="0.12.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
-    <PackageReference Include="Wavefront.AppMetrics.SDK.CSharp" Version="2.4.0" />
+    <PackageReference Include="Wavefront.AppMetrics.SDK.CSharp" Version="3.1.0" />
     <!-- Remove this once AppMetrics SDK version has been updated. -->
-    <PackageReference Include="Wavefront.SDK.CSharp" Version="1.6.0" />
+    <PackageReference Include="Wavefront.SDK.CSharp" Version="1.7.0" />

```

Comment:
let's remove this reference now that AppMetrics SDK is using the latest version of the core SDK